### ISSUE TITLE
Saved image has breaken code structure

### ISSUE THREAD

**Issue by maloletniy** [dawnlabs/carbon#147]
Saved image has invalid line breaks

Save image from (Firefox)
https://carbon.now.sh/?bg=rgba%28171,%20184,%20195,%201%29&t=material&l=auto&ds=false&wc=false&wa=false&pv=0px&ph=0px&ln=true&code={%0A%20%20%22name%22:%20%22test-yarn%22,%0A%20%20%22version%22:%20%221.0.0%22,%0A%20%20%22main%22:%20%22index.js%22,%0A%20%20%22license%22:%20%22MIT%22,%0A%20%20%22scripts%22:%20{%0A%20%20%09%22test%22:%20%22echo%20yarn!%22%0A%20%20}%0A}%0A
## Expected Behavior
correct  rendering of 
```
{
  "name": "test-yarn",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "scripts": {
  	"test": "echo yarn!"
  }
}

```
## Actual Behavior
It moves "test": "echo yarn!" outside the script block
```
{
  "name": "test-yarn",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "scripts": {
  	
  } "test": "echo yarn!"
}
```

**Comment by jakedex** [comment]
![carbon](https://user-images.githubusercontent.com/10369094/33785146-ac8a8ace-dc18-11e7-8053-39aa2d7162ed.png)

Hmm, I'm not able to reproduce the issue on my end. Could you include a few screenshots of the erroneous behavior?

**Comment by maloletniy** [comment]
Hello, on the image you can see how it looks on the page, when I save the image and also I put my firefox version. BTW, on chrome it works correctly.

![2017_12_11_09_00_41_carbon](https://user-images.githubusercontent.com/1572879/33819234-0f325ef0-de52-11e7-9c8d-f81270cb5435.png)

**Comment by JohnPhamous** [comment]
I have a similar problem. Using Firefox:

Here's my input:

```
# Saying hello to 5 different people

def hello(name):
    print("Hello " + name)

for i in range(5):
    userName = input("What is your name? ")
    hello(userName)
```

The output is
![example](https://i.imgur.com/6B9ici4.jpg)

When the preview on the website is
![website](https://i.imgur.com/jVeBvrt.jpg)

**Comment by mfix22** [comment]
@JohnPhamous  @maloletniy, looks like this is just due to us missing some FireFox specific CSS. We are on it.

@jakedex can you take a look when you get the chance?

**Comment by jakedex** [comment]
So it looks like [dom-to-image](https://github.com/tsayen/dom-to-image) has issues loading fonts from external stylesheets. It looks like your browser was falling back to the default, resulting in the weird spacing issues. This should be fixed with #149.
